setup/gcp/installer/util/constants.py

```diff
@@ -104,7 +104,8 @@ class DeploymentStatus(Enum):
     'roles/storage.objectViewer',
     'roles/storage.objectCreator',
     'roles/cloudsql.client',
-    'roles/logging.logWriter'
+    'roles/logging.logWriter',
+    'roles/iam.serviceAccountTokenCreator'
```

Comment:
I'd like to see if this can just be set on the service account instead of the project. Using the SVC_ACCT_ROLES (which should otherwise be deleted as I don't think anything else is using it.)